extensions/interactions/GraphInput/GraphInput.js

```diff
@@ -269,6 +269,7 @@ oppia.directive('graphViz', function() {
               y: $scope.state.mouseY,
               label: ''
             });
+            setMode0(_MODES.MOVE);
```

Comment:
That was my question. What do you think? Do these names accurately describe what the corresponding functions do?

(I'm just going off what you've told me.)